src/Toast/ToastContent.tsx

```diff
@@ -0,0 +1,80 @@
+import * as React from 'react';
+
+import { Box, IBox, IBoxCSS } from '../Box';
+import { Icon, IIcon } from '../Icon';
+import { Text } from '../Text';
+import { ITheme, useTheme } from '../theme';
+import { ToastType } from './index';
+
+/**
+ * TOAST CONTENT
+ */
+
+interface IToastContentProps extends IToastContent, IBox<HTMLElement> {}
+
+export interface IToastContent<T = {}> {
+  title?: string;
+  message?: string;
+  icon?: IIcon['IconProp'];
+  closeIcon?: IIcon['IconProp'] | boolean;
```

Comment:
Looking at actual usage, I don't think boolean is a proper type here.
```
<Icon icon={closeIcon || 'times'}
```

I'd expect that passing `false` would hide the icon.
We can either change the type or handle falsy `closeIcon`.

Reply:
updated so explict false hides the close icon